src/unit-workspace/unit.service.ts

```diff
@@ -0,0 +1,93 @@
+import { Injectable } from '@nestjs/common';
+import { currentDateTime } from '@us-epa-camd/easey-common/utilities/functions';
+
+import { UnitWorkspaceRepository } from './unit.repository';
+import { UnitBaseDTO, UnitDTO } from '../dtos/unit.dto';
+import { EntityManager } from 'typeorm';
+
+@Injectable()
+export class UnitWorkspaceService {
+  constructor(
+    private readonly repository: UnitWorkspaceRepository,
+    private readonly entityManager: EntityManager,
+  ) {}
+
+  async getUnit(locId: string, unitId: number,): Promise<UnitDTO> {
```

Comment:
Why is locId needed?

Reply:
locId is not needed. The Unit Information API for all the others send in the locId as part of the URL structure. I left it for two reasons 1) To make it consistent and 2) if there is ever a need to use the locId for future changes, that change will not impact the UI or all corresponding pipelines (routes, controller, etc).

Reply:
Sounds good.